crates/sui-core/src/authority_active/execution_driver/mod.rs

```diff
@@ -59,6 +59,7 @@ where
         //       transactions awaiting execution, or less often than once per transactions.
         //       However, we need to be sure that if there is an awaiting trasnactions we
         //       will eventually fire the notification and wake up here.
+        tokio::time::sleep(Duration::from_millis(1_000)).await;
```

Comment:
Wonder if 1s too long? In https://github.com/MystenLabs/sui/pull/3718 I tried to distinguish the two cases between where some tx failed and all succeeded: one delays 500ms while the other delays only 100ms

Reply:
Given that this should hardly ever happen, I am happy with the 1s for the moment, and we see

Reply:
Well, but in the happy path (where tx execute successfully), you wouldn't want a 1s delay

Reply:
I recommend you merge the dedup change, and leave the timing control to #3718

Reply:
Ah sorry did not see this in time -- feel free to refactor and remove.